src/gui/properties/speedplotview.h

```diff
@@ -0,0 +1,89 @@
+#ifndef SPEEDPLOTVIEW_H
+#define SPEEDPLOTVIEW_H
+
+#include <QGraphicsView>
+#include <QPen>
+#include <QQueue>
+#include <QMap>
+
```

Comment:
Ok, i understand, i use forward declaration before, but only for fix circular dependency errors.